README.md

```diff
@@ -1 +1,107 @@
+
+You can look at the Google Maps, Mapbox and Leaflet libraries as options to help you with building a map visualization. Try to consider how easy your visualization is to understand. Some concepts which can help to improve the visualization:
+
+* Group clusters of vehicles together
+* Modify your visualization depending on zoom level
+
+We expect your visualization to be accessible via a web browser. It only needs to be tested against the most recent version of Chrome.
+
+## Additional business requirements
+
+If a vehicle has exited the "city boundaries", the system should disregard any GPS emissions from that vehicle. The city boundary is currently assumed to be a 50km radius around door2door's office which is located at latitude 52.53 and longitude 13.403.
+
+All valid position data received must be stored somewhere for future analysis and consumption by the door2door data science team.
+
+## Technical assumptions
+
+* The API is expected to be implemented using one of the following languages: Ruby, Python or Javascript.
+* You are free to make use of any frontend framework or library you please (if any), but you should justify your choice.
+
+## Optional extras
+
+* A containerized solution is not a requirement, but definitely preferred for us to easily deploy your solution and test. Therefore, adding a Dockerfile to your solution is optional, but recommended.
+
+* We expect your solution to be able to handle the load from the provided simulation tool with ease. As an optional extra, you can demonstrate to us how your solution would scale to handle 1000+ vehicles. This could be through a screen capture, or some form of clearly readable performance benchmarking, or in another way if you choose.
+
+## Delivery of your solution
+
+Please deliver your solution to us as a publicly accessible git repository, or in a ZIP file. The repository should contain full instructions for us to run the solution on our own machines.
+
+If you are able to publicly host the solution somewhere (e.g. Heroku, AWS), this would be a great bonus.
+
+## Reviewing
+
+The following description will give you an understanding of how we review the code challenge. What matters to us is to learn how you write code and what you consider as clean code. For us it is more important to have an understandable project than a complex algorithm.
+
+The criteria that we are looking for are the following:
+
+- Documentation: Is the project and the code properly documented?
+- Correctness: Is the task solved? If there is anything missing, is the reason why it is missing documented?
+- Technology: Which libraries or approaches are used? Do they make sense for the task?
+- Code quality: Is the code understandable and maintainable? Does the project make use of Object Oriented Programming? Does the code follow the PEP8 pattern? Is the projected linted?
```

Comment:
Isn't PEP8 specific to Python? Might be confusing for candidates implementing in Ruby/Node